tailwind.config.js

```diff
@@ -98,8 +98,10 @@ module.exports = {
         "1100": "1100px",
         "1435": "1435px",
         "9/10": "90%",
+        "53": "53%",
```

Comment:
I think we should replace this with fraction

Reply:
okeyy done :3